xml/System/Convert.xml

```diff
@@ -18554,11 +18558,12 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="2" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="chars">To be added.</param>
-        <param name="bytes">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="chars">A span containing the string representation that is encoded with base-64 digits.</param>
+        <param name="bytes">When this method returns <see langword="true" />, the converted 8-bit unsigned integers.</param>
```

Comment:
and false?

Reply:
GitHub is acting a bit weird positioning the comments in the correct line.

I assume you are talking about `<param name="bytes">`. I'm looking at the code, and whenever the method returns `false`, the `bytesWritten` param is set to `default` (in other words, zero). I'm not seeing any code that resets the `bytes` parameter to its original value, or resizes it to have a length of zero. Apparently it could contain a part of `chars` already converted, up to the char that caused an error.

I could say: "When this method returns false, either the span remains unmodified (length zero) or contains an incomplete conversion of chars, up to the last valid character."

Reply:
Really weird positioning indeed.